packages/wrangler/src/index.ts

```diff
@@ -71,12 +135,23 @@ import registerTriggersSubcommands from "./triggers";
 import { typesHandler, typesOptions } from "./type-generation";
 import { printWranglerBanner, updateCheck } from "./update-check";
 import { getAuthFromEnv } from "./user";
+import { loginCommand, logoutCommand, whoamiCommand } from "./user/commands";
 import { whoami } from "./user/whoami";
 import { debugLogFilepath } from "./utils/log-file";
 import { vectorize } from "./vectorize/index";
 import registerVersionsSubcommands from "./versions";
 import registerVersionsDeploymentsSubcommands from "./versions/deployments";
 import registerVersionsRollbackCommand from "./versions/rollback";
+import { workflowsInstanceNamespace, workflowsNamespace } from "./workflows";
+import { workflowsDeleteCommand } from "./workflows/commands/delete";
+import { workflowsDescribeCommand } from "./workflows/commands/describe";
+import { workflowsInstancesDescribeCommand } from "./workflows/commands/instances/describe";
+import { workflowsInstancesListCommand } from "./workflows/commands/instances/list";
+import { workflowsInstancesPauseCommand } from "./workflows/commands/instances/pause";
+import { workflowsInstancesResumeCommand } from "./workflows/commands/instances/resume";
+import { workflowsInstancesTerminateCommand } from "./workflows/commands/instances/terminate";
+import { workflowsListCommand } from "./workflows/commands/list";
+import { workflowsTriggerCommand } from "./workflows/commands/trigger";
```

Comment:
There is an argument to be had to hide namespace commands and sub-commands in their own folder, and just define the whole array at once, e.g. so you go from this:

```
registry.define([
		{ command: "wrangler kv:key", definition: kvKeyAlias },
		{ command: "wrangler kv:namespace", definition: kvNamespaceAlias },
	...
	]);
```

To this:
	
```ts
registry.define(kv);
```

But it's worth focussing on the main benefit to having all the registrations in this file as individuals, which is that anyone coming to the codebase has a single place to look to see the entire command API.

It also means you don't have to know what command you're searching for, they are more discoverable if they're laid out here.

Reply:
I like the idea of using a flat structure. 👍🏼